css/style.css

```diff
@@ -236,98 +236,201 @@ body {
     margin-left: -50px;
 }
 
+.profile{
+  background-color: var(--brickRed);
+}
+
 .profilePage_container {
-    background-color: var(--brickRed);
-    height: 100vh;
     color: white;
     font-family: "Fira Mono", monospace;
 }
 
-.profilePage_title p {
-    margin-bottom: 20px;
-    font-weight: bold;
-    font-family: "Fira Mono", monospace;
-    float: right;
-    margin-right: 290px;
-    position: relative;
-    bottom: 290px;
-}
-
 .profilePage_title h1 {
     font-family: "Roboto Slab", serif;
-    font-size: 304px;
-    opacity: 0.1;
+    font-size: 120px;
+    opacity: 0.3;
     text-align: center;
     position: relative;
-    top: -150px;
+    top: 10px;
     text-transform: uppercase;
 }
 
-.profilePage_userDetails h2 {
-    text-transform: uppercase;
-    font-family: "Fira Mono", monospace;
-    margin-left: 40px;
+/* for the bio */
+.profile-bio{
+  position: absolute;
+  right: 780px;
+  width: 300px;
+  height: 400px;
+  top: 50px;
+  /*border: 3px solid green;*/
+}
+
+.bio-card{
```

Comment:
change the class name--- I'm going to stop pointing out the class name changes here but you should go and make the necessary changes for all the remaining class names below.